snyk-monitor/values.yaml

```diff
@@ -158,6 +158,10 @@ sysdig:
   namespace: sysdig-agent
   secretName: sysdig-eve-secret
 
+sysdigV2:
```

Comment:
Would something else make it easier for the customers?
Will it make sense, once the migration is over, for customers to have to configure `sysdigV2` instead of `sysdig`?

Reply:
Option: Remove the need for this altogether and check if the secret was created.